css/styles.css

```diff
@@ -0,0 +1,24 @@
+* {
```

Comment:
@kelseycoughlin use the `body` instead as it is more specific

Reply:
@rmjames fixed it down below!